pkg/utils/validator/resourceoverride_test.go

```diff
@@ -580,3 +711,245 @@ func TestValidateOverridePolicy(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateJSONPatchOverride(t *testing.T) {
+	tests := map[string]struct {
+		jsonPatchOverrides []fleetv1alpha1.JSONPatchOverride
+		wantErrMsg         error
+	}{
```

Comment:
you don't need to repeat the tests about invalid path as you've already covered that in TestValidateJSONPatchOverridePath.

for this func, focus on
- empty jsonPatchOverrides
- nil jsonPathOverrides
- one invalid json path
- invalid remove operation
- valid jsonPathOverrides